Pipfile

```diff
@@ -5,7 +5,6 @@ name = "pypi"
 
 [packages]
 boto3 = "*"
-sentry-sdk = "*"
```

Comment:
Is there any context as to why this is being removed someplace?

Reply:
In our imaginary documentation of the entire pipeline :) but the reason is that using sentry is redundant for this function because it will only ever be called from within the step function (which will send notifications if there is an error in this function).

Reply:
Yeah that's fine. We'd benefit from that context in commit messages in the future both to avoid me being confused and having to ask _and_ to remind us later when someone is trying to add it back in why it's not necessary :)

Reply:
I do intend to include that info in the full pipeline documentation, fwiw.